frontend/src/components/smart/Character.vue

```diff
@@ -0,0 +1,657 @@
+<template>
+   <div class="character-container">
+    <!-- Character Image -->
+    <div class="characterWrapper">
+      <div
+        v-bind:class="
+          ['character-cosmetic-applied-'+
+            characterCosmetics[currentCharacterId],
+            'character-animation-applied-'+
+            characterCosmetics[currentCharacterId]
+          ]
+        "
+      >
+       <div class="animation" />
+      <img
+        class="characterImg placeholder"
+        :src="getCharacterArt(characters[currentCharacterId])"
+        :alt="getCharacterName(currentCharacterId)"
+      />
+      </div>
+    </div>
+    <div class="character-data">
+      <!-- Character Name Heading -->
+      <div class="mb-5 d-flex">
+        <h1 class="title mb-0">{{ getCharacterName(currentCharacterId) }}</h1>
+        <button class="edit-icon" @click="openChangeNameModal" ><img src="../../assets/edit-icon.svg" /></button>
+        <span class="ml-auto text-muted align-self-end">#{{currentCharacterId}}</span>
+      </div>
+      <!-- Character chart info -->
+      <div class="row mb-5 text-white">
+        <div class="col cell">
+          <span class="main-font cell-title">{{$t(`Character.level`)}}</span>
+          <span class="alt-text cell-value">{{ characterLvl }}</span>
+        </div>
+        <div class="col cell">
+          <span class="main-font cell-title">{{$t(`Character.power`)}}</span>
+          <span class="alt-text cell-value">{{ totalCharacterPower }}</span>
+        </div>
+        <div class="col cell">
+          <span class="main-font cell-title">{{$t(`Character.reputation`)}}</span>
+          <span class="cell-value">{{$t(`quests.reputationTier.${ReputationTier[getReputationLevel(reputation)]}`)}}</span>
+        </div>
+        <div class="col cell">
+          <span class="main-font cell-title">{{$t(`Character.element`)}}</span>
+          <p class="alt-text cell-value">
+            <span :class="characterTrait.toLowerCase() + '-icon circle-element'"></span>
+            {{characterTrait}}
```

Comment:
Can all of the above have seperate classes applied to distinguish the data from the value. 


e.g Element   "Earth" coloured green
Level   "1" the one being different

Just generally highlight the data from the text

Reply:
I tried adding a slightly dark background and using primary color for the text so we don't fail on visibility